apps/wallet/src/ui/app/pages/initialize/create/index.tsx

```diff
@@ -6,94 +6,117 @@ import { useNavigate } from 'react-router-dom';
 
 import { createMnemonicValidation } from './validation';
 import Button from '_app/shared/button';
-import CardLayout from '_app/shared/card-layout';
+import { Heading } from '_app/shared/heading';
+import { Text } from '_app/shared/text';
 import ExternalLink from '_components/external-link';
 import Icon, { SuiIcons } from '_components/icon';
 import Loading from '_components/loading';
 import { useAppDispatch } from '_hooks';
 import PasswordFields from '_pages/initialize/shared/password-fields';
 import { createVault } from '_redux/slices/account';
-import { PRIVACY_POLICY_LINK, ToS_LINK } from '_shared/constants';
-
-import st from './Create.module.scss';
+import { ToS_LINK } from '_shared/constants';
 
 const CreatePage = () => {
     const dispatch = useAppDispatch();
     const navigate = useNavigate();
     return (
-        <CardLayout
-            title="Create Password for This Wallet"
-            headerCaption="Create New wallet"
-        >
-            <Formik
-                initialValues={{
-                    terms: false,
-                    password: '',
-                    confirmPassword: '',
-                }}
-                validationSchema={createMnemonicValidation}
-                validateOnMount={true}
-                onSubmit={async (values) => {
-                    try {
-                        await dispatch(
-                            createVault({ password: values.password })
-                        ).unwrap();
-                        navigate('../backup');
-                    } catch (e) {
-                        // Do nothing
-                    }
-                }}
-            >
-                {({ isValid, isSubmitting, errors, touched }) => (
-                    <Form className={st.matchParent}>
-                        <div className={st.matchParent}>
-                            <fieldset
-                                disabled={isSubmitting}
-                                className={st.fieldset}
-                            >
-                                <PasswordFields />
-                                <div className={st.space} />
-                                <label className={st.terms}>
-                                    <Field name="terms" type="checkbox" />
-                                    <span className={st.checkBox}></span>
-                                    <span className={st.checkboxLabel}>
-                                        I read and agreed to the{' '}
-                                        <ExternalLink
-                                            href={ToS_LINK}
-                                            showIcon={false}
+        <div className="flex flex-col flex-nowrap items-center justify-center">
```

Comment:
> Looks good in general, just a few comments.
> 
> There are a few places we need to add some margin:
> 
> <img alt="Screenshot 2023-01-17 at 15 44 18" width="427" src="https://user-images.githubusercontent.com/10210143/212946908-9e3226b2-d35e-405b-8fba-b3d725bdabee.png">
> 
> <img alt="Screenshot 2023-01-17 at 15 44 32" width="427" src="https://user-images.githubusercontent.com/10210143/212946911-0cd45b92-c146-4bae-9496-62708f25305f.png">
> 
> <img alt="Screenshot 2023-01-17 at 15 40 37" width="427" src="https://user-images.githubusercontent.com/10210143/212946873-6688c7f9-cd9f-404c-842b-f96c2105ec8b.png">
> 
> <img alt="Screenshot 2023-01-17 at 15 41 58" width="427" src="https://user-images.githubusercontent.com/10210143/212946890-0f8d5892-ee2f-47dd-b2ce-1f4ac9758c3c.png">
> 
> <img alt="Screenshot 2023-01-17 at 15 42 59" width="427" src="https://user-images.githubusercontent.com/10210143/212946897-a1b0d200-da93-411e-a9ca-c2be904fdbd8.png">
> 
> There are a couple of places that custom arbitrary tailwind values are being used, I think the direction is to avoid them and if we need them add them to the theme.
> 
> Also the checkboxes should be converted to a component. (I am ok to do it in another PR, do we have a ticket for this?)

Yeah, One of the reasons I kept some of the old SCSS. Let me address it